torch/quantization/quantize_fx.py

```diff
@@ -178,11 +178,13 @@ def prepare_fx(
         # optional: specify the path for standalone modules
         # These modules are symbolically traced and quantized as one unit
         "standalone_module_name": [
-           "submodule.standalone"
+           # module_name, qconfig_dict, prepare_custom_config_dict
+           ("submodule.standalone", None, None)
```

Comment:
does `qconfig_dict` == None here means "do not quantize" or "use parent qconfig"?

Reply:
it means "use parent qconfig"
for "do not quantize": qconfit_dict = {"": None}